### ISSUE TITLE
Ctrl+K abuse

### ISSUE THREAD

**Issue by PhilipJFryFAF** [FAForever/fa#2110]
By ctrl+k-ing a unit with a long death animation (t3 power generator/Paragon/etc) you can avoid taking dmg on the unit wreck thus saving the wreck
Easiest example would be a nuke hitting a paragon but the wreck staying alive due to the ctrl+k abues -> which leaves the player with an intact para wreck (200k mass worth)

**Comment by shalkya** [comment]
this is glorious exploit (well done philip !); and extremely game breaking.

**Comment by MrRowey** [comment]
@Garanas this still posible ?

**Comment by speed2CZ** [comment]
It should be possible to keep counting the damage while the death animation plays

**Comment by Garanas** [comment]
Yep, this is still happening. And Speed is right - we can fix it by keeping track of the damage.

**Comment by speed2CZ** [comment]
Worst case scenario could be losing the damage on the tick when the unit is turned into the wreck. But that's just a maybe and it would be pretty hard to time correctly.

**Comment by Garanas** [comment]
That is so precise - if someone manages to do that then they deserve it.
